src/main/java/com/faforever/api/data/domain/AvatarAssignment.java

```diff
@@ -25,26 +33,36 @@
   private Avatar avatar;
 
   @Column(name = "selected")
+  @UpdatePermission(expression = IsEntityOwner.EXPRESSION)
   public boolean isSelected() {
     return selected;
   }
 
   @Column(name = "expires_at")
+  @UpdatePermission(expression = IsModerator.EXPRESSION + " or Prefab.Common.UpdateOnCreate")
   public OffsetDateTime getExpiresAt() {
     return expiresAt;
   }
 
   @ManyToOne(fetch = FetchType.LAZY)
   @JoinColumn(name = "idAvatar")
   @NotNull
+  @UpdatePermission(expression = "Prefab.Common.UpdateOnCreate")
   public Avatar getAvatar() {
     return avatar;
   }
 
   @ManyToOne(fetch = FetchType.LAZY)
   @JoinColumn(name = "idUser")
   @NotNull
+  @UpdatePermission(expression = "Prefab.Common.UpdateOnCreate")
   public Player getPlayer() {
     return player;
   }
+
+  @Override
+  @Transient
+  public Login getEntityOwner() {
```

Comment:
This should be annotated with `com.yahoo.elide.annotation.Exclude`. Best case we could specify in the interface, needs to be tested.

Otherwise, add it to all other implementations as well.

Reply:
How should this manifest? I dont see it in elide responses?

Reply:
Exactly. Should be like @JsonIgnore but for elide.

Reply:
I have no problem adding that but I am curious why since I cannot find entityOwner in AvatarAssigments response. Maybe @Transient is enough or I am blind :)

Reply:
https://github.com/yahoo/elide/blob/ef260b57ef9d8e6a403731dc79a5238bd3fa324c/elide-core/src/main/java/com/yahoo/elide/core/EntityDictionary.java#L111

Reply:
so either show me if you see owner in responses or I dont see much meaning in puting @Exclude anywhere